praktikumsplaner-frontend/frontend/src/router.ts

```diff
@@ -41,6 +42,11 @@ export default new Router({
             name: "getstarted",
             component: GetStarted,
         },
+        {
+            path: "/meldungOertl",
```

Comment:
I would change the path to /meldungAusbilder because /meldungOertl could also be short for /meldungOertlicheAusbildungsleitung